src/Styling/Navbar.css

```diff
@@ -0,0 +1,55 @@
+.Navbar {
+   width: 100vw;
+   height: 200px;
+   margin-top: 10px;
+   position: fixed;
+   display: flex;
+   justify-content: space-between;
+   cursor: pointer;
+}
+.Navbar_logo {
+    align-self: start;
+    margin-left: 20px;
+    cursor: pointer;
+}
+.Menu_bars {
+    width: 80px;
+    margin-right: 20px;
+    transform: rotate(90deg);
+    cursor: pointer;
+}
+
```

Comment:
Consistency in spacing is just as important as following a consistent naming convention. Either add an empty line between each style class/id like you did here, or have no empty lines in between each class/id